### PR TITLE
Add samples and update READMEs

### LINKED ISSUE #274: Create dotnet-watch samples
Show how to do custom dotnet-watch setup's in a sample.

 - watching multiple projects
 - running custom targets with dotnet watch
 - customizing the list of files to be watched

### PR DESCRIPTION
**Pull request by natemcmaster**
We've closed lots of issues recently requesting features that are already possible. These samples show how to harness the power of MSBuild and from dotnet-watch. This should address some of the commonly asked for features such as 

 - allowing dotnet-watch to launch custom processes (like node or python)
 - running multiple commands from dotnet-watch (dotnet test + dotnet run)
 - watching custom files
 - watching multiple projects

Resolves #274

### CODE REVIEW

File: samples/dotnet-watch/WatchJavascriptFiles/Program.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (cesarbs) [inline samples/dotnet-watch/WatchJavascriptFiles/Program.cs]: Do we need copyright headers in samples?

File: samples/dotnet-watch/WatchJavascriptFiles/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Watch javascript files with dotnet-watch
+========================================
+
+## Prerequisites
+
+1. Install .NET Core command line from <https://dot.net/core>
```
Comment (cesarbs) [inline samples/dotnet-watch/WatchJavascriptFiles/README.md]: `1.` seems kinda pointless here.

File: samples/dotnet-watch/WatchJavascriptFiles/Web.csproj [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (cesarbs) [inline samples/dotnet-watch/WatchJavascriptFiles/Web.csproj]: Rename this file to `WatchJavascriptFiles.csproj`?

File: samples/dotnet-watch/WatchMultipleProjects/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Watch multiple projects with dotnet-watch
+=========================================
+
+## Prerequisites
+
+1. Install .NET Core command line from <https://dot.net/core>
```
Comment (cesarbs) [inline samples/dotnet-watch/WatchMultipleProjects/README.md]: Same here.

File: samples/dotnet-watch/WatchJavascriptFiles/Program.cs (hunk 1 above)
Comment (natemcmaster) [inline samples/dotnet-watch/WatchJavascriptFiles/Program.cs]: Probably. Will update

**Comment by natemcmaster** [comment]
:up: :date:

**Pull request merged**